src/pudl/transform/ferc1.py

```diff
@@ -1733,6 +1696,96 @@ def apply_xbrl_calculation_fixes(
                         },
                     },
                 ],
+                "depreciation_expense": [
+                    # Dimension: electric. Temporarily adding metadata only.
+                    {
+                        "calc_component_to_replace": {},
+                        "calc_component_new": {
+                            "name": "depreciation_expense",
+                            "weight": 1.0,
+                            "source_tables": [
+                                "depreciation_amortization_summary_ferc1"
+                            ],
+                            "utility_type": "electric",
+                            "subdimension": "electric",
+                        },
+                    }
+                ],
+                "depreciation_expense_for_asset_retirement_costs": [
+                    # Dimension: electric. Temporarily adding metadata only.
+                    {
+                        "calc_component_to_replace": {},
+                        "calc_component_new": {
+                            "name": "depreciation_expense_asset_retirement",
+                            "weight": 1.0,
+                            "source_tables": [
+                                "depreciation_amortization_summary_ferc1"
+                            ],
+                            "utility_type": "electric",
+                            "subdimension": "electric",
+                        },
+                    }
+                ],
+                "amortization_and_depletion_of_utility_plant": [
+                    {
+                        "calc_component_to_replace": {},
+                        "calc_component_new": {
+                            "name": "amortization_limited_term_electric_plant",
+                            "weight": 1.0,
+                            "source_tables": [
+                                "depreciation_amortization_summary_ferc1"
+                            ],
+                            "utility_type": "electric",
+                            "subdimension": "electric",
+                        },
+                    },
+                    {
+                        "calc_component_to_replace": {},
+                        "calc_component_new": {
+                            "name": "amortization_other_electric_plant",
+                            "weight": 1.0,
+                            "source_tables": [
+                                "depreciation_amortization_summary_ferc1"
+                            ],
+                            "utility_type": "electric",
+                            "subdimension": "electric",
+                        },
+                    },
+                ],
```

Comment:
These calculations link line 8 of the income statement form to the last two data columns in the depreciation amortization summary table.

These factoids appear in the original data table and the original metadata for this table, but they appear to be missing from the exploded metadeta and exploded data, which seems weird to me.  This table is part of the income statement hierarchy that we're working with.

I copied the use of utility type and subdimension here, but I don't understand what subdimension means.

Reply:
@e-belfer does this new calculation seem reasonable?

@cmgosnell do you know why some of these facts might show up in the original data/metadata but not make it through to the exploded versions?

Reply:
Re the data/metadata - do you mean that the calculations themselves originally show up in the metadata? I haven't seen subdimensional calculations anywhere in the original metadata so it'd be good to know if that's the case and I've been missing something.